config/env.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+module.exports = {
+  dotEnv: function() {
```

Comment:
Maybe this can be something more descriptive, like `configureEnvironmentVariables`?

Reply:
Thanks for the feedback. Addressed in ae7d63a.